### DISCUSSION TITLE
Updating Amazon Linux 2023 dnf packages

### DISCUSSION THREAD

**Discussion by long-wan-ep** [renovatebot/renovate#29461] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab, renovate 37.385.0

### Please tell us more about your question or problem

Hi, I'm wondering if anyone has a solution for using renovate to update packages on amazon linux 2023, installed via `dnf`. We have dockerfiles using AL2023 as the base image, which we install packages such as python 3.11, we're currently using regex manager and `docker` as the datasource, but AL2023 does not keep up to date with the latest python 3.11 tags released on dockerhub.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by secustor** [comment]
You can check if the [`repology`](https://docs.renovatebot.com/modules/datasource/repology/) datasource works for you.

**Reply by long-wan-ep** [reply]
Thanks for the suggestion, I looked at repology but AL2023 is not listed.

**Reply by amimas** [reply]
Any other suggestions? As @long-wan-ep mentioned, Amazon Linux 2023 is not listed in repology.

**Reply by secustor** [reply]
If no other source has the same version the remaining option is to contribute or sponsor a RPM datasource.

**Reply by amimas** [reply]
Yeah, I agree. I don't know about where and what should be contributed. Not sure what you meant by sponsoring though.

**Reply by secustor** [reply]
Take a look at the [datasource folder](https://github.com/renovatebot/renovate/tree/main/lib/modules/datasource), there a new RPM datasource would have to be added. Also this WIP PR for a debian datasource could be helpful. https://github.com/renovatebot/renovate/pull/30071 

[With sponsoring I have meant paying an external person to contribute the implementation for you.](https://github.com/renovatebot/renovate/blob/main/CODE_OF_CONDUCT.md#getting-more-help)

**Reply by nejch** [reply]
This could potentially also be solved in Repology, but AWS would need to provide some package links upstream first (see discussions in https://github.com/repology/repology-updater/pull/1320).
